manifest.gradle

```diff
@@ -1,7 +1,7 @@
 ext {
     mavRepoInternalUrl = System.getenv('LOCAL_MAVEN_URL')
 
-    github_username = 'gotev'
+    github_username = 'biosensics'
```

Comment:
why are we modifying this?

Reply:
This is used somewhere in the address. So github.com/gotev/ -> should be github.com/biosensics/ so that we fetch from correct repo.